### PR TITLE
Fix Vm build.

### PR DESCRIPTION
**Pull request by sleirsgoevy**

### CODE REVIEW

File: setup_db.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+(
+    id SERIAL PRIMARY KEY NOT NULL,
+    toolchain_id VARCHAR(100) NOT NULL,
+    state  submission_state NOT NULL
+);
+
+CREATE UNIQUE INDEX submissions_submission_id_uindex ON submissions (submission_id);
+CREATE INDEX submissions_state_index ON submissions (state); -- optimizes invoker queries
+
+COMMENT ON TABLE submissions IS 'Contains information on all submissions';
```
Comment (MikailBag) [inline setup_db.sql]: setup_db.sql was excluded.
To create DB scheme, all migrations (db/migrations/*/up.sql) should be applied

File: init-jjs-root.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+EOF
+mkdir "$1/etc/toolchains"
+cat > "$1/etc/toolchains/cpp.toml" << EOF
+name="cpp"
+suffix="cpp"
+[[build]]
+argv=["/usr/bin/g++", "\$(System.SourceFilePath)", "-o", "\$(System.BinaryFilePath)", "-std=c++17", "-Wall", "-Wextra", "-Wpedantic", "-DJJS"]
+[run]
+argv=["\$(System.BinaryFilePath)"]
+EOF
```
Comment (MikailBag) [inline init-jjs-root.sh]: Why is it better thar init-jjs-root tool?

Comment (sleirsgoevy) [reply init-jjs-root.sh]: That piece of shit doesnt work for some reason. Also I think shell scripts are more appropriate for this task.

Comment (MikailBag) [reply init-jjs-root.sh]: But anyway, config shouldn't be hardcoded.
What *should* be done:
```
if has_python()
    soft --with=python
    cp example-config/toolchains/python.toml JJS_ROOT/etc/toolchains/python.toml
if has_gcc()
    soft --with=gcc
    cp example-config/toolchains/gcc.toml JJS_ROOT/etc/toolchains/gcc.toml
```
and so on.
I'm now working on soft to teach in how to install `python` and `gcc`, but if you want you can rewrite it

File: devtool/src/main.rs [hunk 3]
```diff
@@ -167,6 +167,19 @@ fn task_package() {
         format!("{}/bin/jjs-db-init", &pkg_dir),
     )
     .unwrap();
+    let opts = fs_extra::dir::CopyOptions {
```
Comment (MikailBag) [inline devtool/src/main.rs]: why not fs_extra::dir::CopyOptions::new()?

Comment (sleirsgoevy) [reply devtool/src/main.rs]: Copy-pasted from later code.

**Pull request merged**